### PR TITLE
Remove homedir references

### PR DESCRIPTION
**Pull request by obino**

### CODE REVIEW

**Comment by obino** [comment]
OCD run https://ocd.appscale.com:8080/job/Daily%20Build/929/

File: AppController/lib/helperfunctions.rb [hunk 1]
```diff
@@ -175,7 +175,7 @@ def self.read_json_file(location)
 
   # Extracts the version from the VERSION file.
   def self.get_appscale_version
-    version_contents = self.read_file(APPSCALE_HOME + '/VERSION')
+    version_contents = self.read_file(APPSCALE_CONFIG_DIR + '/VERSION')
```
Comment (cdonati) [inline AppController/lib/helperfunctions.rb]: I don't think this change will work since VERSION is not in `.appscale`

Comment (cdonati) [inline AppController/lib/helperfunctions.rb]: Actually, ignore that. I didn't realize it was copied in.

File: AppController/terminate.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 $:.unshift File.join(File.dirname(__FILE__), "lib")
-APPSCALE_HOME = ENV['APPSCALE_HOME']
+CONFIG_FILE_LOCATION = "/etc/appscale"
```
Comment (menivaitsi) [inline AppController/terminate.rb]: Why not use the same variable name (APPSCALE_CONFIG_DIR) as above?

Comment (obino) [inline AppController/terminate.rb]: We have different names in different places. Djinn uses CONFIG_FILE_LOCATION ... I could try to uniform the name if you think this PR is good for it.

Comment (menivaitsi) [inline AppController/terminate.rb]: Well, this is a new constant, so use the same name; please and thank you.

Comment (obino) [inline AppController/terminate.rb]: Roger that: I change the variaable to APPSCALE_CONFIG_DIR everywhere I found it.

**Comment by obino** [comment]
OCD run https://ocd.appscale.com:8080/job/Daily%20Build/932/

**Comment by obino** [comment]
OCD run https://ocd.appscale.com:8080/job/Daily%20Build/934/

**Comment by obino** [comment]
OCD run https://ocd.appscale.com:8080/job/Daily%20Build/935/

**Pull request merged**